docs/experimental-features/security-posture-management.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[security-posture-management]]
+== Cloud security posture management
+
+NOTE: This feature requires Elastic Stack version 8.3.0 or later.
+
+Cloud security posture management (CSPM) and Kubernetes security posture management (KSPM) continuously monitor and compare your cloud and kubernetes infrastructure against security best practices in order to help you identify and remediate misconfigurations. 
+
+[discrete]
+=== Enable security posture management
+
+The steps to enable this feature differ between self-hosted and Elastic cloud deployments.
+
+[[enable-kspm-on-cloud]]
+[discrete]
+==== Enable KSPM for cloud deployments
+
+First, enable the KSPM flag for your deployment:
+
+* From the left-side navigation menu in Kibana, click on **Manage this deployment**.
+
+image::images/kspm-1.png[The Manage deployment button]
+
+* Under "My deployment", select **Kibana**.
+* Under "Instances", click the three-dot menu next to your instance, then click **Edit configuration**.
+* Click **Edit user settings** in the upper right of the page.
```

Comment:
Is this for Kibana?  You can click on similarly hyperlinked text for Elasticsearch and Enterprise Search.
<img width="809" alt="Screen Shot 2022-06-24 at 3 02 54 PM" src="https://user-images.githubusercontent.com/79124755/175648974-1fbe8eb1-d603-4afd-9bd3-711c31c15da5.png">


<img width="815" alt="Screen Shot 2022-06-24 at 3 03 03 PM" src="https://user-images.githubusercontent.com/79124755/175648990-a1f0e388-7ffa-4af2-87f5-2a8a1d36a98e.png">

Reply:
Yes I think it's got to be the Kibana user settings. Does the way it's written seem like it could mean to click "Edit user settings" next to Enterprise search or Elasticsearch instead?

Reply:
Yep, I would adjust it like so:

. In the Kibana section, click **Edit user settings**.